stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/attachments/AttachmentState.kt

```diff
@@ -18,16 +18,52 @@ package io.getstream.chat.android.compose.state.messages.attachments
 
 import io.getstream.chat.android.client.models.Message
 import io.getstream.chat.android.compose.state.imagepreview.ImagePreviewResult
+import io.getstream.chat.android.compose.state.mediagallerypreview.MediaGalleryPreviewResult
 
 /**
  * Represents the state of Attachment items, used to render and add handlers required for the attachment to work.
  *
  * @param message Data that represents the message information.
  * @param onLongItemClick Handler for a long click on the message item.
  * @param onImagePreviewResult Handler when the user selects an action to scroll to and focus an image.
+ * @param onMediaGalleryPreviewResult Handler used when the user selects an action to perform from
+ * [io.getstream.chat.android.compose.ui.attachments.preview.MediaGalleryPreviewActivity].
  */
-public data class AttachmentState(
+public data class AttachmentState
+@Deprecated(
+    message = "Constructor containing parameter 'onImagePreviewResult' has been deprecated, " +
+        "please use the constructor that does not have said parameter. Parameter 'onMediaPreviewResult' " +
+        "is the replacement for the parameter 'onImagePreviewResult' and is functionally the same.",
+    level = DeprecationLevel.WARNING
+)
+constructor(
     val message: Message,
     val onLongItemClick: (Message) -> Unit = {},
+    @Deprecated(
+        message = "This parameter has been deprecated. Replace it with" +
+            "'AttachmentState.onMediaPreview'.",
+        level = DeprecationLevel.WARNING
+    )
     val onImagePreviewResult: (ImagePreviewResult?) -> Unit = {},
-)
+    val onMediaGalleryPreviewResult: (MediaGalleryPreviewResult?) -> Unit,
```

Comment:
should we have this as a default value = {} given that this way it's much more breaking? And the previous result had an empty lambda for default, so no reason not to have it?

Reply:
Actually, that's a good point since this is a leftover I accidentally forgot.
Basically, I did not initially provide a default value so I could track the change more easily via compiler errors.

Adding it now.